src/Gmres_Polyk.kokkos.cxx

```diff
@@ -482,20 +482,12 @@ PETSC_INTERN void mat_mult_powers_share_sparsity_kokkos(Mat *input_mat, const in
    aijkok_local_output->transpose_updated = PETSC_FALSE;
    aijkok_local_output->hermitian_updated = PETSC_FALSE;
    // Invalidate diagonals
-   Mat_SeqAIJ *a = (Mat_SeqAIJ *)mat_local_output->data;
```

Comment:
Having a look through the changes in the PETSc MR, am I correct in thinking the diagonal no longer has to be invalidated for kokkos matrices when the entries change?